test/correctness/atomics.cpp

```diff
@@ -1174,7 +1174,7 @@ void test_async_tuple(const Backend &backend) {
 
     // Run 10 times to make sure race condition do not happen
     for (int iter = 0; iter < 10; iter++) {
-        Realization out = consumer1.realize(2 * img_size);
+        Realization out = consumer1.realize({2} * img_size);
```

Comment:
?

Reply:
Yikes. My RegExp-fu for fixing these failed. Let me manually recheck them all.

Reply:
Yeah, I missed a *lot* of stuff. Oops.